software/speller/data_collection_platform/dcp/mp/shared.py

```diff
@@ -19,3 +19,8 @@
 
 # communicate whether the BCI device is ready (connected)
 is_bci_ready = Value("i", 0)
+
+# creating a manager that will allow processes to be communicated to from the flask app
+# the manager will control when processes can pull from the bci device
+manager = Manager()
+bci_running = manager.dict()
```

Comment:
@wykzhang I think this variable name could be changed for `bci_processes_states` or something, because we are going to store more information than if a bci process is running or not. Otherwise everything looks good and we are good to merge!

Reply:
Agreed. Will change it now